pkgs/shells/zsh/zsh-fast-syntax-highlighting/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "zsh-fast-syntax-highlighting";
+  version = "2019-07-20";
+
+  src = fetchFromGitHub {
+    owner = "zdharma";
+    repo = "fast-syntax-highlighting";
+    rev = "ab78177ee23a6509be7e8fe09d265d5640fb333a";
+    sha256 = "1h65ydb90wzl0f2w0q3sgc77l1zdn6w33fv8fg5nvx2ycl2xr43x";
+  };
+
+  #adapted from https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=zsh-fast-syntax-highlighting-git
+  installPhase = ''
+    _plugindir="$out/share/zsh/site-functions"
+    _chromadir="$_plugindir/chroma"
+    _themesdir="$_plugindir/themes"
+
+    cd "${src}"
+    install -dm0755 "$_plugindir"
+    install -m0644 fast-* "$_plugindir"
+    install -m0644 _fast-* "$_plugindir"
+    install -m0644 -- -fast-* "$_plugindir"
+
+    cd "./chroma"
+    install -dm0755 "$_chromadir"
+    install -m0644 -- *.ch "$_chromadir"
+
+    cd "../themes"
+    install -dm0755 "$_themesdir"
+    install -m0644 -- *.ini "$_themesdir"
+  '';
+
+  meta = with lib; {
+    description = "Syntax-highlighting for Zshell";
+    homepage = https://github.com/zdharma/fast-syntax-highlighting ;
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ qyliss ];
```

Comment:
```suggestion
```

I would not be a good maintainer of this software, considering I’ve never heard of it!